services/background-worker/test/Test/Wire/BackendNotificationPusherSpec.hs

```diff
@@ -326,7 +327,7 @@ spec = do
           http2Manager = undefined
           statuses = undefined
           rabbitmqAdminClient = Just $ mockRabbitMqAdminClient mockAdmin
-          rabbitmqVHost = "test-vhost"
+          amqpEP = Q.AmqpEndpoint undefined undefined "test-vhost" undefined
```

Comment:
Can't the whole `amqpEP` be `undefined` (instead of having almost al fields of `AmqpEndpoint` being undefined)? :thinking:

Reply:
Finally, it's not really important. :smile: 

One could also ask if `AppT` isn't too big if we only need the `vhost`. Or, if `getRemoteDomains :: RabbitMqAdmin.AdminAPI (Servant.AsClientT IO) -> Text {- vhosy -} -> AppT IO [Domain]` wouldn't be sufficient. Hower, not our construction place right now :wink: